### ISSUE TITLE
recipientList get misdocumented

### ISSUE THREAD

**Issue by WanderingBrooks** [SparkPost/node-sparkpost#215]
Either the [implementation](https://github.com/SparkPost/node-sparkpost/blob/master/lib/recipientLists.js) or [documentation/example](https://github.com/SparkPost/node-sparkpost/blob/master/examples/recipientLists/get.js#L7) is incorrect. It seems a small change to the documentation/example like this 
``` 
var options = {
    show_recipients: true
}
// Promise
 client.recipientLists.get('UNIQUE_TEST_ID', options, callback function )
  .then(data => {
   console.log('Congrats you can use our client library!');
   console.log(data);
   })
   .catch(err => {
   console.log('Whoops! Something went wrong');
   console.log(err);
   });
```
Would fix the issue and make it easier to understand how to use that aspect of the library.

**Comment by aydrian** [comment]
Sorry for the confusion. It looks like I might have missed that when I was updating the examples for v2 of the client library. We'll get that corrected as soon as we can. Thank you for bringing this to our attention.

**Comment by aydrian** [comment]
Closed by #216
